app/screens/home/channel_list/categories_list/index.tsx

```diff
@@ -72,6 +73,7 @@ const CategoriesList = ({hasChannels, iconPad, isCRTEnabled, moreThanOneTeam}: C
                         shouldHighlighActive={true}
                     />
                 }
+                <DraftsButtton/>
```

Comment:
I wonder if this should only mount if there are any drafts

Reply:
The mounting is actually handled inside the component. Return null if there is not draft.

Reply:
Okay got it. Will change it.